src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -1109,9 +1102,6 @@ NOINLINE freeze_result FreezeBase::recurse_freeze_stub_frame(frame& f, frame& ca
     // native frame
     return freeze_pinned_native;
   }
-  if (UNLIKELY(ContinuationHelper::CompiledFrame::is_owning_locks(_cont.thread(), &map, senderf))) {
```

Comment:
Are there still other calls to is_owning_locks or can it be removed?

Reply:
Yes, it's also called for CONT_isPinned0. It can be removed too now that I look at it.

Reply:
too bad, I thought they could be removed but at least they aren't platform dependent.

Reply:
Yeah, we need them for verification of the lock-counting mechanism.